### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jlid87**
Updates to the content including:

Changing references to 'discovery' to 'alpha'.
Creating migration guide, API reference and error code pages.
Creating and populating pages on endpoints:

Submit a new draft excise movement
Get excise movements
Submit a message for an excise movement
Get all messages for an excise movement
Get trader information

API reference page is a placeholder and will be replaced with Redocly page. Have uploaded application.yaml to test this.

### CODE REVIEW

File: .DS_Store
Comment (ChrisCookOC) [inline .DS_Store:1]: Do you need to commit these .DS_Store files? It's Mac specific metadata thing https://en.wikipedia.org/wiki/.DS_Store

Comment (jlid87) [reply .DS_Store:1]: Probably not @ChrisCookOC, good spot!

File: source/documentation/submit-draft-movement.html.md.erb [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+## Successful response
+
+If you use this endpoint to send a small message (up to 10MB), a successful response will contain:
+
+- a movement ID
+- a consignor ID
+- a local reference number (LRN)
+- a status
+
+It may also contain an administrative reference code (ARC).
```
Comment (mauro-fanton) [inline source/documentation/submit-draft-movement.html.md.erb:21]: I thought that the submit draft move is an async process. By that I mean if you their is an error you get an error  response back straight away, but if the trader is successful he/she will get a 202 (acceped) code back and then the trader would need to pull for the ARC.

Comment (jlid87) [reply source/documentation/submit-draft-movement.html.md.erb:21]: @mauro-fanton could be! The yaml file had a response schema (attached screenshot) that included these items so I included them in the service guide. But I did think it looked like dummy data - would be strange to have a 'Cancelled' response to a draft movement submission, for example.

It depends how accurate we want the information to be for alpha / user testing. I could include a line at the top that makes it clear this text doesn't reflect the finished API and will change as development progresses.

Screenshot 2023-08-18 at 13 49 57

**Review by mauro-fanton — APPROVED** [review]

**Pull request merged**